modules/server/src/network/__tests__/index.test.ts

```diff
@@ -0,0 +1,122 @@
+// @ts-nocheck
+// graphql types have readonly modifiers all the way down
```

Comment:
disabling TS on this file. It's a test file so it won't break the build. 
It's a larger piece of work to type because all the graphql lib types have `readonly` modifiers , which means util funcs like `pick` and `omit` don't work. The full types are too broad as we're using a specific subset of the functionality.